src/main/scala/fundamentals/level02/ListExercises.scala

```diff
@@ -85,6 +85,18 @@ object ListExercises {
     */
   def showListSize[A](xs: List[A]): String = ???
 
+  /**
+    * scala> val people = List(Person("Tokyo", 30), Person("Berlin", 43), Person("Moscow", 50), Person("The Professor", 80))
+    * scala> sortByAgeDescending(people)
+    * = List(Person("The Professor", 80), Person("Moscow", 50), Person("Berlin", 43), Person("Tokyo", 30))
+    *
+    * Hint: `sortBy` takes a function from `Person` to the field that you want to sort by.
+    * The field must have some sort of `Ordering` defined.
+    * All common types, e.g. `Int`, `Long`, `Float`, `Double`, `String` have an `Ordering` defined.
+    */
+  def sortByAgeDescending(unsortedPeople: List[Person]): List[Person] =
+    unsortedPeople.sortBy(???)
+
```

Comment:
Can we change ```unsortedPeople.sortBy(???)``` to ```???```. Otherwise it is giving compile error.

Reply:
Ah thanks. We need CI for this, haha.